assets/newspack-ui/scss/variables/_spacing.scss

```diff
@@ -0,0 +1,12 @@
+:root {
+	// Spacing - not ems because the font size of the element itself caused a cascade
+	--newspack-ui-spacer-1: 8px;
```

Comment:
Suggestion: change this to `--newspack-ui-spacer-base` for better readability when tweaking per context (in a modal or media queries)

Reply:
Something like this:

```css
--newspack-ui-spacer-base: 8px;
--newspack-ui-spacer-1: var( --newspack-ui-spacer-base );
--newspack-ui-spacer-2: calc( var( --newspack-ui-spacer-base ) * 1.5 ); // 12px
...etc
```